samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+
+
+const { TranscriptionProperties } = require('speech_to_text_api_v30');
+var SpeechToTextApiV30 = require('speech_to_text_api_v30');
+var sleep = require('system-sleep');
+var defaultClient = SpeechToTextApiV30.ApiClient.instance;
+const request = require('request');
+
+   //Create API Instance
+   var apiInstance = new SpeechToTextApiV30.DefaultApi();
+// Your subscription key and region for the speech service
+const SUBSCRIPTION_KEY = ""
+
+const SERVICE_REGION = "centralindia"
+
+const NAME = "Simple transcription"
+const DESCRIPTION = "Simple transcription description"
+
+const LOCALE = "en-US"
+const RECORDINGS_BLOB_URI = ""
+
+// Provide the uri of a container with audio files for transcribing all of them with a single request
+const RECORDINGS_CONTAINER_URI = ""
+
+//Set model information when doing transcription with custom models
```

Comment:
nit: please always put a single space after `//`

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.